### PR TITLE
lyft helper

### PR DESCRIPTION
**Pull request by cpruijsen**
note: private logic (paths, headers, body parameters) all git ignored.

### CODE REVIEW

File: lyft-helper.js [hunk 1]
```diff
@@ -1,26 +1,60 @@
-/* TODO:
--
--
--
+var lyftMethods = require('lyftPrivateMethods');
```
Comment (daredia) [inline lyft-helper.js]: this doesnt work, does it? i fixed a bug earlier where you had syntax like this, but i think it needs to be `require('./lyftPrivateMethods')` right?

File: lyft-helper.js [hunk 2]
```diff
@@ -1,26 +1,60 @@
...
+  }).catch( function(err) {
+    console.log('error post of phoneCodeAuth LYFT', err);
+  });
 };
 
-var lyftPhoneCodeAuth = function(fourDigitCode) {
-  // user inputs 4 digit code, we fetch(url, {method: POST});
-  // we take response and store in lyftPrivateMethods ?
-  // lyftMethods.phoneCodeAuthHeaders, lyftMethods.phoneCodeAuthBody (function)
+var getCost = function(token, session, endAddress, endLat, endLng, startLat, startLng, startAddress) {
```
Comment (daredia) [inline lyft-helper.js]: may want to collapse those address, lat, and long variables into one object that gets passed

**Comment by daredia** [comment]
@cpruijsen this looks pretty good so far, i assume you're not ready for me to merge this yet though right?

File: lyft-helper.js (hunk 1 above)
Comment (cpruijsen) [inline lyft-helper.js]: oops - fixing in the next commit / PR.

File: lyft-helper.js (hunk 2 above)
Comment (cpruijsen) [inline lyft-helper.js]: sounds good. merge when you want (so you can have a local copy).

File: lyft-helper.js (hunk 1 above)
Comment (cpruijsen) [inline lyft-helper.js]: fixed.

File: lyft-helper.js (hunk 2 above)
Comment (cpruijsen) [inline lyft-helper.js]: done

File: uber-helper.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+var fetch = require('node-fetch');
+var uberMethods = require('./uberPrivateMethods');
+var baseURL = 'http://cn-sjc1.uber.com'; // https ?
```
Comment (daredia) [inline uber-helper.js:3]: note - uber helper is unfinished and untested so far

**Pull request merged**